core/src/main/java/com/xatkit/core/recognition/IntentRecognitionProviderFactory.java

```diff
@@ -76,12 +78,19 @@ public static IntentRecognitionProvider getIntentRecognitionProvider(@NonNull Xa
 
         IntentRecognitionProvider provider;
 
+        // TODO Add a new key in the configuration to choose the recognition provider and change if/else to switch
```

Comment:
Remove this comment, we have discussed it internally.
I'll take care of it.

Reply:
Yes you right!
Thank you!